settings.gradle

```diff
@@ -15,4 +15,5 @@ if (target.isCommon()) {
     include ':sample-linuxx64-app'
     include ':rxjava2-interop'
     include ':rxjava3-interop'
+    include ':retrofit'
```

Comment:
I'm not sure that this is a good module name, @CherryPerry what do you think?

Reply:
Yes, I think `retrofit-adapter` will be much better.